src/index.js

```diff
@@ -16,7 +16,49 @@ import leaveList from './modifiers/leaveList';
 import insertText from './modifiers/insertText';
 import createLinkDecorator from './decorators/link';
 import createImageDecorator from './decorators/image';
+import { addText, addEmptyBlock } from './utils';
 
+const INLINE_STYLE_CHARACTERS = [' ', '*', '_'];
+
+function checkCharacterForState(editorState, character) {
+  if (INLINE_STYLE_CHARACTERS.indexOf(character) === -1) { return editorState; }
```

Comment:
This line is not covered yet. [ref](https://102-75265088-gh.circle-artifacts.com/0/tmp/circle-artifacts.xcy0Hit/coverage/src/index.js.html)